server/controllers/interviewController.js

```diff
@@ -21,4 +22,15 @@ module.exports = {
     await interviewDoc.save(); // what constitutes a completed interview?
     res.json(interviewDoc); // Feedback submitted? Review period ended?
   },
+  joinInterview: async (req, res) => {
+    try {
+      const interview = await Interview.findOne({ _id: req.params.id });
+      await interview.users.push(req.user);
+      await interview.save();
+      await req.user.save();
```

Comment:
Should we be doing something like `await users.interviews.push(interview)` before saving the user?